packages/123done/static/components/normalize.css

```diff
@@ -228,7 +228,7 @@ code,
 kbd,
 pre,
 samp {
-  font-family: monospace, monospace;
+  font-family: monospace;
```

Comment:
Duplicated font-family threw an error, but if "normalize.css" is a 3rd party file, maybe we should exclude it from linting via [_dev/.stylelintrc](https://github.com/mozilla/fxa/pull/13307/files#diff-000f24f3c6e292cf234c59821eb225c1e02784db5b8512f06860d51aef881aff) `ignoreFiles[]`.